src/app/component/result/result.component.ts

```diff
@@ -8,7 +8,7 @@ import quizData from 'src/assets/json/data.json';
   styleUrls: ['./result.component.css'],
 })
 export class ResultComponent implements OnInit, OnDestroy {
-  quizData: any = { ...quizData };
+  quizData = { ...quizData };
   isQuizCompleted: any;
   userName: any;
```

Comment:
Could you please check and remove userName if not in  use

Reply:
Yes i removed it.